src/zope/server/buffers.py

```diff
@@ -14,9 +14,9 @@
 """Buffers
 """
 try:
-    from cStringIO import StringIO
+    from cStringIO import StringIO as BytesIO
 except ImportError:
-    from StringIO import StringIO
+    from io import BytesIO
```

Comment:
Probably not!  I'll try it and see how it goes.

Reply:
Here's how it goes: all tests pass.  But since test coverage is 78%, ¯\\_(ツ)_/¯

(That's partially why I preferred sticking with cStringIO on Python 2.7: it might not work on Python 3, but at least it wouldn't introduce regressions on Python 2.)

Reply:
We're pretty close to 100% coverage. I can work on a followup PR to try to get us there.